client.lua

```diff
@@ -339,19 +361,25 @@ RegisterCommand("drugs-N2O", function(source, args, rawCommand)
 	end
 end)
 
-RegisterCommand("killhigh", function(source, args, rawCommand)
-	if kh == true then
+RegisterCommand("killdrugs", function(source, args, rawCommand)
+	if kd == true then
+		high = false

```

Comment:
fixes your bug with the killhigh. It allows users to use other drugs directly after.